tests/model/test_model.py

```diff
@@ -1407,8 +1408,9 @@ def test__handle_typing_event(self, mocker, model,
     ], ids=[
         'remove_19', 'add_30'
     ])
-    def test__handle_subscription_event(self, model, mocker, stream_button,
-                                        event, final_muted_streams):
+    def test__handle_subscription_event_muting(self, model, mocker,
+                                               stream_button, event,
+                                               final_muted_streams):
```

Comment:
Is this intentional?

Reply:
Yes, this test deals only with muting, I'll move this to the previous commit though